CHANGELOG-prerelease.md

```diff
@@ -1,3 +1,9 @@
+## Prerelease 37 ( TBD )
+
+Tag: TBD
```

Comment:
```suggestion
Tag: [v1.0.0-prerelease.37](https://github.com/patternfly/patternfly-elements/releases/tag/v1.0.0-prerelease.37)
```

Reply:
Sure, sounds good.  I didn't add anything there since I didn't actually know if this PR will be merged by the time 37 goes out.